backend/server.py

```diff
@@ -243,6 +243,12 @@ def api_instruction():
     return make_response(c['instruction'], 200)
 
 
+@app.route('/images/<path:filename>')
```

Comment:
Maybe we can support any file types?

Reply:
Good point. I've included audio types as well checking when filetypes match current config.

Reply:
I'd like to rename /images/ to /assets/ or something like and proxy any type of files over this